main/politeness_logi.R

```diff
@@ -0,0 +1,43 @@
+library(pscl)
+library(plyr)
+library(readr)
+
+# get data
+dat <- read.csv("/dev/stdin")
+dim(dat)
```

Comment:
this is a noop, right?

Reply:
I found out that R will print them out to stdout. And since I used `sink()` to redirect stdout, it will go to the file. But I don't need it to go to the file. I've changed it to an `assert`.